### DISCUSSION TITLE
Can't display any avatar (even images from README.md)

### DISCUSSION THREAD

**Discussion by SwiftBeta** [community/community#24718] [Apps, API and Webhooks]
<p>Since some days ago I can’t display any avatar in any repo. The following image is a screenshot from the Alamofire repo (<a href="https://github.com/Alamofire/Alamofire" class="inline-onebox" rel="noopener nofollow ugc">GitHub - Alamofire/Alamofire: Elegant HTTP Networking in Swift</a>). And as you can see I’m not able to load any image, including the ones from the README.md.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/3/232e093cf97e2418b713c8baa428f4e0ef2d10ea.png" data-download-href="/uploads/short-url/51dlhKjmj50qGgsYmORuh3bhwL0.png?dl=1" title="Screen Shot 2021-08-06 at 08.36.05" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/74316958/181086445-90af0895-b3bc-4569-ab1a-ec8b6583e07b.png" alt="Screen Shot 2021-08-06 at 08.36.05" data-base62-sha1="51dlhKjmj50qGgsYmORuh3bhwL0" width="690" height="349" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/3/232e093cf97e2418b713c8baa428f4e0ef2d10ea_2_690x349.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/3/232e093cf97e2418b713c8baa428f4e0ef2d10ea_2_1035x523.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/2/3/232e093cf97e2418b713c8baa428f4e0ef2d10ea_2_1380x698.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/3/232e093cf97e2418b713c8baa428f4e0ef2d10ea_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2021-08-06 at 08.36.05</span><span class="informations">1917×971 202 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<ul>
<li>I have cleaned my cache</li>
<li>If I use another browser: Safari, Chrome, etc I have the same error (even if I’m not logged in)</li>
<li>Same happens when I use the browser from my mobile (without been logged)</li>
</ul>
<p>Do you have any clue?</p>
<p>thanks in advance!</p>

**Comment by tajmone** [comment, marked as answer]
<aside class="quote no-group" data-username="SwiftBeta" data-post="1" data-topic="194091">
<div class="title">
<div class="quote-controls"></div>
 SwiftBeta:</div>
<blockquote>
<p>Do you have any clue?</p>
</blockquote>
</aside>
<p>I’ve been experiencing this and similar problems (especially CSS not loading correctly, showing  broken page layout) on and off, intermittently. Some days the problem seems more frequents, others less, but I’ve never experienced it as a permanent problem (usually refreshing the page one or more times solved it).</p>
<p>If you’re experiencing this constantly, you might try and test if using a PROXY or an alternative DNS fixes it, in that case the problem could be with your ISP or default DNS settings.</p>
